src/components/filmPopups/FilmDeletePopup.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {FormEvent} from "react";
+import { Bg, BgForm, Form, BtnPopupClose, BtnSubmit } from "./filmPopups.styled";
+import { Container } from "../../styles/general";
+import {useDeleteMovieContext} from "../../context/deleteMovieContext";
+
+
+const FilmDeletePopup = () => {
+  let { isDeletePopupShown, setDeletePopupShown } = useDeleteMovieContext();
+
+  const closeDeletePopup = (e: { preventDefault: () => void;}) => {
```

Comment:
As opposed to the form submit event, there's nothing dangerous in default action triggered upon click event to prevent that, so you could go as simple, as:
```js
const closeDeletePopup = () => 
    setDeletePopupShown(!isDeletePopupShown)
```